man/git-delete-branch.md

```diff
@@ -3,29 +3,44 @@ git-delete-branch(1) -- Delete branches
 
 ## SYNOPSIS
 
-`git-delete-branch` &lt;branchname&gt;
+`git-delete-branch` [-r &lt;remote[:remote branch name]&gt;] &lt;branchname&gt;
 
 ## DESCRIPTION
 
-  Deletes local and remote branch named &lt;branchname&gt;.
-  Note that local deletion fails if the branch is checked out.
+Delete local and remote branches. If the remote switch (-r) is provided, then
+use the given remote and remote branch. Otherwise, the remote branch is found
+by first trying the upstream tracking branch. If there is no tracking branch
+configured, then deletion will not succeed unless provided explicitly. If the
+remote branch and local branch do not share a name, the remote branch name can
+be included with the remote switch.
+
+Local deletion will fail if the branch is currently checked out. If the local
+operation fails, then the remote operation will not run.
 
 ## OPTIONS
 
-  &lt;branchname&gt;
+-r
 
-  The name of the branch to delete.
-  If multiple branches are provided, then they will all be deleted.
+Specify a remote name and a remote branch name to use during the remote
+branch deletion.
```

Comment:
At my first glance, it seems we need to provide two arguments, one is remote name and the other is remote branch name. Maybe we could mention the format of arguments to make it clearer.